deploy/role.yaml

```diff
@@ -157,3 +159,16 @@ rules:
   - '*'
   verbs:
   - '*'
+- apiGroups:
```

Comment:
why is this needed?

Reply:
These are to avoid the following error:

{"level":"error","ts":1568832561.2466843,"logger":"ctrl.addon.addon install","msg":"failed to apply release.yaml","Request.Namespace":"","Request.NamespaceName":"/dashboard","Request.Name":"dashboard","error":"clusterroles.rbac.authorization.k8s.io \"tekton-dashboard-minimal\\
" is forbidden: attempt to grant extra privileges:

[{[use] [security.openshift.io] [securitycontextconstraints] [] []}
{[get] [extensions] [ingresses] [] []} {[list] [extensions] [ingresses] [] []}
{[watch] [extensions] [ingresses] [] []}
{[get] [apps] [ingresses] [] []}
{[list] [apps] [ingresses] [] []}
{[watch] [apps] [ingresses] [] []}
{[get] [dashboard.tekton.dev] [extensions] [] []}
{[list] [dashboard.tekton.dev] [extensions] [] []}
{[create] [dashboard.tekton.dev] [extensions] [] []}
{[update] [dashboard.tekton.dev] [extensions] [] []}
{[delete] [dashboard.tekton.dev] [extensions] [] []}
{[patch] [dashboard.tekton.dev] [extensions] [] []}
{[watch] [dashboard.tekton.dev] [extensions] [] []}]